br/pkg/restore/range.go

```diff
@@ -28,36 +26,25 @@ func SortRanges(ranges []rtree.Range, rewriteRules *RewriteRules) ([]rtree.Range
 	rangeTree := rtree.NewRangeTree()
 	for _, rg := range ranges {
 		if rewriteRules != nil {
-			startID := tablecodec.DecodeTableID(rg.StartKey)
```

Comment:
We don't need rewrite rules at all ?

Reply:
I'm not sure for now. Maybe need rewrite rule in the future. So keep to rewrite rule placement and just remove the table id encoding/decoding parts.